components/common/Card.tsx

```diff
@@ -15,7 +15,7 @@ const styles = StyleSheet.create({
   card: {
     backgroundColor: 'white',
     borderRadius: 10,
-    shadowColor: Colors.primary[500],
```

Comment:
I use this card in other places as well. Can you take this change back? You can always pass the style you wanted via the `style` prop.